modules/core/include/visp3/core/vpNetwork.h

```diff
@@ -45,6 +45,7 @@
 #include <iostream>
 #include <stdio.h>
 #include <string.h>
+#include <sys/select.h>
```

Comment:
Error was something like:

```
visp/modules/core/include/visp3/core/vpNetwork.h:128:3: error: 'fd_set' does not name a type
  128 |   fd_set readFileDescriptor;
      |   ^~~~~~
```

Should be needed from the [doc](https://linux.die.net/man/3/fd_set). Let see if the build is ok on CI.